geodesy/package.xml

```diff
@@ -25,17 +26,24 @@
 
   <run_depend>geographic_msgs</run_depend>
   <run_depend>geometry_msgs</run_depend>
-  <run_depend>python-pyproj</run_depend>
   <run_depend>tf</run_depend>
   <run_depend>sensor_msgs</run_depend>
   <run_depend>unique_id</run_depend>
   <run_depend>uuid_msgs</run_depend>
 
+  <run_depend condition="$ROS_PYTHON_VERSION == 2">python-pyproj</run_depend>
+  <run_depend condition="$ROS_PYTHON_VERSION == 3">python3-pyproj</run_depend>
+
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_depend>
+  <buildtool_export_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_export_depend>
+
   <!-- these dependencies are only for testing -->
   <test_depend>rosunit</test_depend>
 
   <!-- documentation dependencies -->
-  <build_depend>python-catkin-pkg</build_depend>
+  <build_depend condition="$ROS_PYTHON_VERSION == 2">python-catkin-pkg</build_depend>
+  <build_depend condition="$ROS_PYTHON_VERSION == 3">python3-catkin-pkg</build_depend>
```

Comment:
I'd highly recommend changing this to `python-catkin-pkg-modules` and `python3-catkin-pkg-modules`. The reason is those packages are side-by-side installable, while `python-catkin-pkg` and `python3-catkin-pkg` conflict because they both provide the same CLI tool. This mostly matters on Ubuntu Bionic where users may have both ROS Melodic and ROS Eloquent installed because they target different Python versions.

Reply:
Will do